### DISCUSSION TITLE
Stable version release

### DISCUSSION THREAD

**Discussion by devher0** [cubefs/cubefs#2609] [Ideas]
Hi guys can you prepare stable version, because v3.2.0  stopped working problem with data nodes and released version  v.3.3.0 also doesnt work the problem in some os gcc modules

**Comment by leonrayang** [comment]
> Hi guys can you prepare stable version, because v3.2.0  stopped working problem with data nodes and released version v.3.3.0 also doesnt work the problem in some os gcc modules

@devher0  Please provide the error message so that we can proceed with fixing it. Thank you.
Here is some case of compile issues, https://cubefs.io/docs/master/faq/build.html
